app/services/booking_responder/reject.rb

```diff
@@ -1,44 +1,9 @@
 class BookingResponder
```

Comment:
When rejecting all the visitors and selecting a slot, the slot granted persists in the DB even thought the visit is rejected